<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayListSettings</class>
 <widget class="QWidget" name="PlayListSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*background-color: rgb(42, 42, 42);*/

QLabel
{
	color: rgb(216, 216, 216);
}

QPushButton 
{	
	color: rgb(216, 216, 216);
	background-color: rgb(0, 0, 0);
	border-radius: 10px;
}

QPushButton:hover 
{
	background-color: rgb(35, 11, 63);
}

QPushButton:disabled
{
	color: rgb(73, 73, 73);
}

QPushButton:pressed 
{
	background-color: rgb(82, 26, 149);
}

QTableWidget
{
	background-color: rgb(0, 0, 0);
}

QTableWidget::item::selected
{
	background-color: rgb(82, 26, 149);
}

QHeaderView::section
{
	background-color: rgb(20, 6, 36);
}

QHeaderView
{
	background-color: rgb(30, 30, 30);
}</string>
  </property>
  <widget class="QWidget" name="Titlebar_widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>781</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(27, 27, 27);</string>
   </property>
   <widget class="QLabel" name="Titlebar_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>PlayList Settings</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft</set>
    </property>
   </widget>
  </widget>
  <widget class="QTableWidget" name="PlayList_table">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>611</width>
     <height>371</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(216, 216, 216);
</string>
   </property>
   <property name="autoScroll">
    <bool>false</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
   </property>
   <property name="columnCount">
    <number>2</number>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>250</number>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>PlayList Name</string>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Source</string>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="Edit_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string> Edit        </string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/white-edit.png</normaloff>:/icons/icons/white-edit.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="Back_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>  Back     </string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/white-left-arrow.png</normaloff>:/icons/icons/white-left-arrow.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>15</width>
     <height>15</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="Delete_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>  Delete       </string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/white-minus.png</normaloff>:/icons/icons/white-minus.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="AddNew_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Add New       </string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/white-plus-sign.png</normaloff>:/icons/icons/white-plus-sign.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="background_widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>801</width>
     <height>421</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0.369, y1:0.182, x2:1, y2:1, stop:0 rgba(57, 9, 126, 255), stop:1 rgba(129, 18, 129, 255));</string>
   </property>
  </widget>
  <zorder>background_widget</zorder>
  <zorder>Titlebar_widget</zorder>
  <zorder>PlayList_table</zorder>
  <zorder>Edit_button</zorder>
  <zorder>Back_button</zorder>
  <zorder>Delete_button</zorder>
  <zorder>AddNew_button</zorder>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
